plutus-e2e-tests/README.md

```diff
@@ -9,8 +9,16 @@ This framework is still in early stages of development with only a handful of te
 - Spending funds locked at script using reference script, reference inputs and providing datum as witness in txbody.
 - Minting tokens using reference script and providing script witness in txbody.
 
+It is also possible to run these tests on a public testnet, see preconditions:
+- Have a directory containing at least these two directories:
+  - "utxo-keys" containing "test.skey" and "test.vkey" files. These must both be text envelope PaymentKey format, currently no support for PaymentExtendedKey.
+  - "ipc" containing the active node.socket
+- Cardano node is fully synced on a public network (e.g. preview testnet)
+- There is at least one ada-only UTxO at the test account. Each test will spend a few ada so make sure there's enough funds.
+- Modify the test options passed to each test to be run from `TestnetOptions` to `LocalNodeOptions`. E.g. swap `testnetOptionsBabbage8` for `localNodeOptionsPreview`.
+- Ensure the `LocalNodeOption`'s `localEnvDir` points to the directory containing your keys ("utxo-keys") and node socket ("ipc").
+
```

Comment:
Is it so that the test.skey and .vkey I simply create myself (with `cardano-cli keygen`?), then trasfer ada to it via a public faucet?

Reply:
Yes. `cardano-cli address key-gen` with default key type `normal-key`, will produce two text envelope files, `PaymentSigningKeyShelley_ed25519` and `PaymentVerificationKeyShelley_ed25519` that can be used. Can get funds from faucet or other wallet.